package.json

```diff
@@ -45,24 +45,15 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
-  "browser": {
```

Comment:
Thanks for that! However, this is still needed, otherwise asynchronous methods won't work in the browser.